docs/index.txt

```diff
@@ -79,8 +79,21 @@ Using Gradle:
 Alternatively you can follow the instructions on the Bintray repository
 overview page by clicking the "Set me up!" button.
 
+
+.. _jdbc_connecting_to_crate:
+
+Connecting to CrateDB server
+============================
+
+CrateDB >= 2.0.x enables `Host-Based Authentication`_ by default. Clients that
```

Comment:
"2.0"

Reply:
It's "2.1",  2.0 has HBA disabled

Reply:
ok, I thought it was always enabled by default since begin. Pushed a fixup.